packages/merge/src/typedefs-mergers/enum.ts

```diff
@@ -18,7 +18,7 @@ export function mergeEnum(
           : 'EnumTypeExtension',
       loc: e1.loc,
       directives: mergeDirectives(e1.directives, e2.directives, config),
-      values: mergeEnumValues(e1.values, e2.values, config),
+      values: mergeEnumValues(e2.values, e1.values, config),
```

Comment:
It appears that the enum type and value mergers are implemented with reversed argument order from each other. While the enum type merger assigns `A -> B`, the enum value merger assigns `B -> A`. That means you'll end up with type settings from one candidate and value settings from the other... :sigh:

This seems like the least invasive correction to make these align.

Reply:
Makes sense to me for now, might make sense to open up separate issue on that for consistency, defer to @ardatan

Reply:
https://github.com/ardatan/graphql-tools/issues/2464